generators/app/templates/hooks/pre-commit

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
You'll only know it's there when you're trying to commit a file with a `console.log` in it.